### PR TITLE
Update the contributing.md

### PR DESCRIPTION
**Pull request by Himanshunitrr**
I was the first to find the typo in contributing.md(issue no. #1523) but some bankebihariagrawal copied it, but copied it in a wrong way.

Fixes #

#### Describe the changes you have made in this PR -
Fixed typo in contributing.md
### Screenshots of the changes (If any) -

Note: Please check **Allow edits from maintainers.** if you would like us to assist in the PR.

### CODE REVIEW

**Comment by Himanshunitrr** [comment]
I am new to GitHub, therefore waited for some time for comment in issue #1521 but later realized that for minor issues one has to create a pull request as mentioned in contributing.md

File: CONTRIBUTING.md [hunk 1]
```diff
@@ -4,7 +4,7 @@
 Thanks for checking out our contribution guide. We greatly appreciate any work contributed, no matter how small!
 
 ## How to Contribute
-When you are ready to start work on an issue:
+When you are ready to start, work on an issue:
```
Comment (tachyons) [inline CONTRIBUTING.md:7]: Intentional as this is a full sentence

Comment (Himanshunitrr) [reply CONTRIBUTING.md:7]: But what about #1521 sir. I noticed it first and some other guy created a PR

Comment (tachyons) [reply CONTRIBUTING.md:7]: Person who created the issue and person who solved the issue need not be the same, `an` was needed, `,` isn't

Comment (Himanshunitrr) [reply CONTRIBUTING.md:7]: I thought that one to has to create an issue and waited. Then I came to know that for typos one has to create a PR.
I suggested "an" too. You can have look at #1521 .
Yes sir I agree that person who created the issue and person who solved the issue need not be the same, but it was my mistake that I didn't create a PR,

**Pull request closed without merging**